easydeasy_rest/api/serializers.py

```diff
@@ -33,7 +28,25 @@ class Meta:
 
 
 class ArticleSerializer(ModelSerializer):
+
+    def create(self, validated_data):
+        print(validated_data)
+
+        # Updates the to generic image if possible
+        if validated_data['img'] is None:
+            # Gets First Disease in list
+            disease: Disease = validated_data['diseases'][0]
+
+            image_folder = '/images/' + disease.category.name + '/'
+            folder_path = settings.MEDIA_ROOT + image_folder
```

Comment:
it will cause problems on different OS's. 
should either use pathlib Path object to create the path or os.path to make it compatible for every OS.

same for image_folder.

Reply:
Not sure about this since the django server will only run on a single operating system (probably linux) so technaclly it can have this path name like this